javascript-light-bulb/main.js

```diff
@@ -0,0 +1,13 @@
+const $clickSwitch = document.querySelector('.switch');
+const $container = document.querySelector('.container');
+$clickSwitch.addEventListener('click', handleSwitch);
+
+function handleSwitch(event) {
+  if ($clickSwitch.className === 'switch') {
+    $clickSwitch.className = 'switch ' + 'dark';
```

Comment:
Can also do this without the `+`.

`$clickSwitch.className = 'switch dark'`

Reply:
Good to know. Thank you !